stetho/src/main/java/com/facebook/stetho/inspector/elements/AbstractChainedDescriptor.java

```diff
@@ -155,4 +155,14 @@ public final void getStyles(Object element, StyleAccumulator accumulator) {
 
   protected void onGetStyles(E element, StyleAccumulator accumulator) {
   }
+
+  @Override
+  @SuppressWarnings("unchecked")
+  public final void getAccessibilityNodeInfo(Object element, StyleAccumulator accumulator) {
```

Comment:
maybe call this `getAccessibilityStyles` so it matches up with `getStyles`, and because we're still using `StyleAccumulator`. (and then follow up with the cascading renames elsewhere of course)